objectivec/src/ort_enums.mm

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "ort_enums_internal.h"
+
+#include <algorithm>
+
+#include "core/session/onnxruntime_cxx_api.h"
+
+namespace {
+struct LoggingLevelInfo {
+  ORTLoggingLevel logging_level;
+  OrtLoggingLevel capi_logging_level;
+};
```

Comment:
Are there any other options for disambiguating ORTLoggingLevel and OrtLoggingLevel?

Reply:
none that come to mind now